HadesArchitect/grafana-cassandra-source
# Repack

General update, plugin compliance, better readability, build automation.

**What was done**
- Restructured source code
- Removed SimpleJson
- Added Grunt
- Improved Readme
- Added license
- Added plugin required files
- Added logo

**How to test**
- Review the changeset
- Clone locally 
- Remove old node_modules and docker containers
- Build & launch the project following the README.md instructions
- Check for warnings in logs (should be no warnings)
- In grafana logs should be `msg="Registering plugin" logger=plugins name="Apache Cassandra"`
- Check that grafana is running and the plugin is available in a local DS list
